<launch>
	<arg name="lat_origin" default="0.0" />
	<arg name="lon_origin" default="0.0" />
	<node pkg="helper_nodes" type="odometry" name="odometry" output="screen">
		<param name="origin_latitude" value="$(arg lat_origin)" />
		<param name="origin_longitude" value="$(arg lon_origin)" />
		<param name="K_NS" value="111120.00" />
	</node>
</launch>
